packages/graphql/src/schemaTypes/objectTypes/gql-Subscription.ts

```diff
@@ -0,0 +1,21 @@
+import { subscriptionType } from 'nexus'
+import { DevState } from './gql-DevState'
+import { Query } from './gql-Query'
+
+export const Subscription = subscriptionType({
+  definition (t) {
+    t.field('authChange', {
+      type: Query,
+      description: 'Triggered when the auth state changes',
+      subscribe: (source, args, ctx) => ctx.emitter.subscribeTo('authChange'),
+      resolve: (source, args, ctx) => ({}),
+    })
+
+    t.field('devChange', {
+      type: DevState,
+      description: 'Issued for internal development changes',
+      subscribe: (source, args, ctx) => ctx.emitter.subscribeTo('devChange'),
+      resolve: (source, args, ctx) => ctx.coreData.dev,
+    })
+  },
```

Comment:
Just to make sure I'm seeing the light at the end of the tunnel; we will subscribe to something like:

```ts
t.field('config', {
  type: 'JSON',
  description: 'resolved cypress.config.js for current project',
  subscribie: (_, __, ctx) => ctx.emitter.subscribeTo('configChange'),
  resolve: (_, __, ctx) => ctx.lifecycleManager.fullConfig
})
```

So we can reactively update/change the UI when the config changes (eg re-run a spec, etc)?

We can also use this to tell the runner to re-run a spec, for example when someone changes + saves the current spec and the filesystem watcher triggers?

Reply:
Right I see, same idea